chroma_core/south_migrations/0040_table_update_notify.py

```diff
@@ -61,13 +61,13 @@ def fill_template(template):
 """)
 
 backward_trigger_template = fill_template("""
-DROP TRIGGER %(name)s_notify_update
+DROP TRIGGER IF EXISTS %(name)s_notify_update
 ON %(name)s;
 
-DROP TRIGGER %(name)s_notify_insert
+DROP TRIGGER IF EXISTS %(name)s_notify_insert
 ON %(name)s;
 
-DROP TRIGGER %(name)s_notify_delete
+DROP TRIGGER IF EXISTS %(name)s_notify_delete
```

Comment:
Is this transition new since the last release?  Should we worry about this not taking affect?

Reply:
It's being called by 0047 below, so it be run.